better document cron options (#439)
temporalio/documentation: docs/php/workflows.md

All interface methods must have one of the following annotations:

- **#[WorkflowMethod]** indicates an entry point to a Workflow.

>   It contains parameters such as timeouts and a task queue.

Reviewer: ```suggestion
  It contains parameters that specify timeouts and a Task Queue name.
```

var_dump($run->getExecution()->getID());
```

### Recurring start

> You can start your workflows on a regular schedule with [the CronSchedule option](distributed-cron).

Reviewer: ```suggestion
You can start a Workflow Execution on a regular schedule with [the CronSchedule option](distributed-cron).
```
